spec/features/user_browses_loan_requests_spec.rb

```diff
@@ -5,33 +5,52 @@
                             password: "pass",
                             name: "jorge")
   }
-  let(:create_item) { LoanRequest.create(title: "Farm Tools",
+  let (:user2) { User.create(email: "jeff@example.com",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.